File: ClientApp/src/app/report/entity/shipping.ts
export class Shipping {

  public id !: number;
  public shopnumber !: string;
  public root !: string;
  public num !: string;


  constructor(id: number, shopnumber: string, root: string, num: string) {
    this.id = id;
    this.shopnumber = shopnumber;
    this.root = root;
    this.num = num;
  }
}

File: ServerApp/src/main/java/lk/cooperative/cooperativejaela/report/entity/Shipping.java
package lk.cooperative.cooperativejaela.report.entity;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class Shipping {
    @Id
    private Integer id;
    private String shopnumber;
    private String root;

    private String num;


    public Shipping() {  }


    public Shipping(String shopnumber, String root, String num) {
        this.shopnumber = shopnumber;
        this.root = root;
        this.num = num;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getShopnumber() {
        return shopnumber;
    }

    public void setShopnumber(String shopnumber) {
        this.shopnumber = shopnumber;
    }

    public String getRoot() {
        return root;
    }

    public void setRoot(String root) {
        this.root = root;
    }

    public String getNum() {
        return num;
    }

    public void setNum(String num) {
        this.num = num;
    }
}
